config/puma.rb

```diff
@@ -5,7 +5,7 @@
 preload_app!
 
 rackup      DefaultRackup
-port        3000
+port        ENV['PORT']      || 3000
```

Comment:
Style/StringLiterals: Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.